scripts/getPVAliases.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+# execute python script
+/cds/group/pcds/pyps/conda/py39/envs/pcds-5.8.2/bin/python "$ENG_TOOLS_SCRIPTS/getPVAliases.py" "$@"
```

Comment:
Two things here:

- Do you know where `$ENG_TOOLS_SCRIPTS` gets set? Mine doesn't even match the engineering tools on my path, so for me this script actually will run a different .py file than the one I intend!
- Maybe it'd be more portable to have this point to the same-directory .py file instead of using the custom environment variables

```suggestion
THIS_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
/cds/group/pcds/pyps/conda/py39/envs/pcds-5.8.2/bin/python "$THIS_DIR/getPVAliases.py" "$@"
```